pkg/cmd/server/api/validation/oauth.go

```diff
@@ -102,6 +103,17 @@ func ValidateOAuthConfig(config *api.OAuthConfig, fldPath *field.Path) Validatio
 			)))
 	}
 
+	if config.TokenConfig.AccessTokenTimeoutSeconds != nil {
```

Comment:
`if timeout := config.TokenConfig.AccessTokenTimeoutSeconds; timeout != nil && *timeout != 0 {` is a bit easier to read.

Reply:
ok